nomad/structs/volumes.go

```diff
@@ -0,0 +1,188 @@
+package structs
+
+import (
+	"github.com/mitchellh/copystructure"
+	"github.com/mitchellh/mapstructure"
+)
+
+const (
+	VolumeTypeHost = "host"
+)
+
+// ClientHostVolumeConfig is used to configure access to host paths on a Nomad Client
+type ClientHostVolumeConfig struct {
+	Name     string `hcl:",key"`
+	Source   string `hcl:"source"`
```

Comment:
I find it a bit confusing that `Source` here means path on host (e.g. `/tmp`) while source in the job spec means the name of volume instead (e.g. `tmp-dir` in PR example).  Given that there is no `destination` here, I'd consider calling it `path` instead?

Reply:
Would've been nice to get that comment in the RFC, but sure :woman_shrugging: